WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -24,21 +24,38 @@
 - [ ] Verify that you can delete a bookmark in edit mode
 - [ ] Verify that you can delete a bookmark folder with bookmarks inside
 - [ ] Verify adding a bookmark domain, subpaths is retained and you are successfully able to visit the domain subpath in a new tab
+- [ ] Verify adding bookmark for many open tabs works. Ensured opening all bookmarks in `Saved tabs` folder will load all the tabs
+- [ ] Verify import/export bookmarks works
 
 ## Favourites
 
 - [ ] Verify editing favourites and changing their URLs updates the favicons accordingly
 - [ ] Verify that you can remove favourites
 - [ ] Verify that you can add new favourites from the `Share with...` menu
+- [ ] Verify adding Favorites Widget to home screen and ensure favorites are updated on the widget accordingly
 
 ## Context menus
 
 - [ ] Make sure context menu items in the URL bar work
 - [ ] Make sure context menu items on content work with no selected text
 - [ ] Make sure context menu items on content work with selected text
 - [ ] Make sure context menu items on content work inside an editable control (input, textarea, or contenteditable)
-- [ ] Verify you can open links in a new tab or private tab. Ensure you can open links in background tabs.
-
+- [ ] Verify you can open links in a new tab or private tab. Ensure you can open links in background tabs
+- [ ] Verify all actions work in context menu when long pressing on `+` in the tool bar
+- [ ] Verify all actions work in context menu when long pressing on tab icon
+- [ ] Verify `Recently Closed Tabs` is opened when long pressing `+` icon in tab view
+
+## Navigation
+
+- [ ] Verify search works from the search box on NTP
+- [ ] Verify searching using voice search icon
+- [ ] Verify changing search engine in settings will update search queries for regular and private tabs
+- [ ] Verify tapping on arrow icon for the suggested searches will only populate that search query in the search box
+- [ ] Verify scanning QR code from the search box will higlight scanned url
+- [ ] Verify all actions in tool bar using Top Bar mode
+- [ ] Verify searching page content using `Find in Page`
```

Comment:
Suggestion

`Verify searching page content using Find in Page works as expected`

Reply:
I added 'works as expected' to most of the ones you mentioned since that is your preference. I think theres a bit of a tradeoff between keeping test cases concise and readability/instruction.